templates/ords-remix-jwt-sample/Database/Modules/concert_app.euser.v1.sql

```diff
@@ -399,18 +399,8 @@ BEGIN
       p_source         => 
 'SELECT * FROM MUSIC_GENRES');
 
-    
-    
   l_modules(1) := 'concert_app.euser.v1';
 
-  ORDS.DEFINE_PRIVILEGE(
-      p_privilege_name => 'concert_app_euser',
-      p_roles          => l_roles,
-      p_patterns       => l_patterns,
-      p_modules        => l_modules,
-      p_label          => 'Non authenticated end user privilege',
-      p_description    => 'Provides limited access to the concert app endpoints',
-      p_comments       => NULL);
```

Comment:
did you export the module definition from the Modules view in DBActions? find it weird to have the `l_roles`, `l_modules` and `l_patterns` variables defined if they are not used.

Reply:
Change implemented now.